api/src/middlewares/services/EmailService/NotifyService.ts

```diff
@@ -82,12 +82,17 @@ export class NotifyService implements EmailServiceProvider {
       ...(additionalContexts[country as string] ?? {}),
       ...(additionalContexts[post as string] ?? {}),
     };
-    return Object.keys(template).reduce((acc, curr) => {
+    const mapPersonalisationValuesToKeys = this.mapPersonalisationValues(personalisationValues);
+    return Object.entries(template).reduce(mapPersonalisationValuesToKeys, {});
```

Comment:
what are you reducing the template entries `to`?

`.reduce(toPersonalisation)`?